sklearn/neighbors/regression.py

```diff
@@ -291,9 +291,28 @@ def predict(self, X):
             y_pred = np.array([np.mean(_y[ind, :], axis=0)
```

Comment:
Do we want this to be issuing a warning in this case? I think the tests should assert either that a warning is raised, or that none is.

Reply:
I have made a test now that asserts *no* warning is raised.